File: sthlmtraveling/src/main/java/com/markupartist/sthlmtraveling/provider/ApiConf.java
package com.markupartist.sthlmtraveling.provider;

import com.markupartist.sthlmtraveling.AppConfig;

public class ApiConf {
   public static String KEY = AppConfig.STHLM_TRAVELING_API_KEY;

   public static String apiEndpoint2() {
       return AppConfig.STHLM_TRAVELING_API_ENDPOINT;
   }

   public static String get(String s) {
       return s;
   }

}

File: sthlmtraveling/src/main/java/com/markupartist/sthlmtraveling/provider/ApiConf.kt
package com.markupartist.sthlmtraveling.provider

import com.markupartist.sthlmtraveling.AppConfig

object ApiConf {
    @JvmField
    var KEY: String = AppConfig.STHLM_TRAVELING_API_KEY

    fun apiEndpoint2(): String {
        return AppConfig.STHLM_TRAVELING_API_ENDPOINT
    }

    @JvmStatic
    fun get(s: String?): String? {
        return s
    }
}
